Sources/SwifterSwift/SwiftUI/ViewExtensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ViewExtensions.swift
+//  SwifterSwift
+//
+//  Created by Jevon Mao on 6/18/21.
+//  Copyright © 2021 SwifterSwift. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
+public extension View {
+    
+    /**
+     Modifier to erase a SwiftUI `View` type into `AnyView`
+     
+     An `AnyView` allows changing the type of view used in a given view hierarchy. Whenever the type of view used with an AnyView changes, the old hierarchy is destroyed and a new hierarchy is created for the new type.
+     */
+    @inlinable func eraseToAnyView() -> AnyView {
```

Comment:
Why is this `@inlinable`? Is that required? Do you know for sure that it's useful/necessary to have here? I think in general we don't use it in the project, but we never decided a standard for if/when to do so.

Reply:
I can't explain the details of `@inlinable` (I think it helps compiler optimizes function call in some cases), I copied this from my own project and the `@inlinable` is just there. Should I remove it for simplicity?